.github/workflows/ecobalyse-data-sync.yml

```diff
@@ -0,0 +1,20 @@
+name: ecobalyse-data sync
+
+on:
+  pull_request:
+    branches: [ master, staging ]
+  workflow_dispatch:
+
+jobs:
+  check-ecobalyse-data-sync:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install wget
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y wget
```

Comment:
I’m pretty sure this is not needed as you can do the same with `curl -O` and curl should be available, no?

Reply:
done